content/cloud-computing/concepts/faas/faas.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: 'FaaS'
+Description: 'Function as a service (FaaS) is a cloud computing model that offers the ability to develop applications without building and maintaining the typical infrastructure needed for application development and deployment.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Cloud'
+  - 'Functions'
+  - 'Development'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Function as a service (FaaS)** is a cloud computing model that offers the ability to develop applications without building and maintaining the typical infrastructure needed for application development and deployment. The application functionality is developed and hosted on the cloud resulting in a "serverless" architecture.
```

Comment:
I don't think the term "serverless" is made clear here. Maybe mention like "The lack of needing to configure and maintain servers for these architecture has led to them being called "serverless"

Reply:
How does this read/sound?

```suggestion
**Function as a service (FaaS)** is a cloud computing model that offers the ability to develop applications without building and maintaining the typical infrastructure needed for application development and deployment. The application functionality is developed and hosted on the cloud. This removes the need for the configuration and maintenance of a server, resulting in a "serverless" architecture.
```

Reply:
This would bring the first use of "server" at the beginning of the entry, hence the new Docs link.